### PR TITLE
Feature/header fillet

### PR DESCRIPTION
**Pull request by CardosoAdriel**

### CODE REVIEW

File: src/modules/shared/components/molecules/Fillet/styles.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    font-size: var(--fs-body-small);
+    font-weight: var(--fw-body-small);
+    line-height: var(--lh-body-small);
+  }
+`;
+
+export const Fillet = styled.section`
+  background-color: var(--primary-300);
+  color: var(--white);
+  height: 72px;
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/styles.ts:20]: Evite usar pixel como unidade de medida. Use sempre tamanhos relativos para projetos que visam responsividade. Nesse caso, use `rem`.

File: src/modules/shared/components/molecules/Fillet/styles.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  &.hidden {
+    display: none;
+  }
+`;
+
+export const WrapFillet = styled.div`
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/styles.ts:27]: Aqui, entendo que sua intenção é se referir a um elemento que vá envolver (a wrapper) outros elementos, nesse caso, chamamos de **wrapper** (envelope, invólucro) ou **container**.

File: src/modules/shared/components/molecules/Fillet/styles.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  max-width: 1440px;
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/styles.ts:28]: Mesma regrinha dos tamanhos relativos.

File: src/modules/shared/components/molecules/Fillet/styles.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  margin: auto;
+  display: flex;
+  justify-content: space-around;
+  padding: 2rem;
+`;
+
+export const WrapContentFillet = styled.div`
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/styles.ts:35]: Evite usar tags não semânticas, como `div`. Procure sempre trabalhar com semântica quando possível, isso é fundamental para SEO e Acessibilidade. Nesse caso, `section`.

File: src/modules/shared/components/molecules/Fillet/styles.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  display: flex;
+  align-items: center;
+  gap: 20px;
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/styles.ts:38]: Tamanhos relativos.

File: src/modules/shared/components/molecules/Fillet/index.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import { Text, Fillet, WrapContentFillet, WrapFillet } from "./styles";
+import { textsFillet } from "@shared/constants/fillet";
+import { useMediaQuery } from "@mui/material";
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import DateRangeIcon from "@mui/icons-material/DateRange";
+import PhoneIcon from "@mui/icons-material/Phone";
+
+export default function FilletWrapper() {
```
[the remark this answers is not in the document]
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/index.tsx:8]: Corrected

File: src/modules/shared/components/molecules/Fillet/index.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  const matches = useMediaQuery("(max-width: 700px)");
+  return (
```
[the remark this answers is not in the document]
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/index.tsx:10]: Corrected

File: src/modules/shared/components/molecules/Fillet/styles.ts (hunk 1 above)
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/styles.ts:20]: Corrected

File: src/modules/shared/components/molecules/Fillet/index.tsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    <Fillet className={matches ? "hidden" : ""}>
+      <WrapFillet>
+        {textsFillet.map((item) => (
+          <WrapContentFillet key={item.id}>
+            {item.id === 1 ? (
```
[the remark this answers is not in the document]
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/index.tsx:15]: Corrected

File: src/modules/shared/components/molecules/Fillet/styles.ts (hunk 2 above)
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/styles.ts:27]: Corrected

File: src/modules/shared/components/molecules/Fillet/styles.ts (hunk 3 above)
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/styles.ts:28]: Corrected

File: src/modules/shared/components/molecules/Fillet/styles.ts (hunk 4 above)
Comment (CardosoAdriel) [reply src/modules/shared/components/molecules/Fillet/styles.ts:35]: Corrected

File: src/modules/shared/components/molecules/Fillet/index.tsx [hunk 9]
```diff
@@ -1,28 +1,25 @@
...
 
-export default function FilletWrapper() {
+export default function Fillet() {
   const matches = useMediaQuery("(max-width: 700px)");
+
   return (
-    <Fillet className={matches ? "hidden" : ""}>
-      <WrapFillet>
+    <FilletContainer className={matches ? "hidden" : ""}>
+      <WraperFillet>
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/index.tsx:15]: Aqui vão dois pontos:

- "Wrapper", com dois p
- Components são como classes, tbm temos que manter padrão. Nesse caso, se vc usou `FilletContainer`, use `FilletWrapper`

File: src/modules/shared/components/molecules/Fillet/styles.ts [hunk 10]
```diff
@@ -14,27 +14,28 @@ export const Text = styled(RawText)`
...
+  display: flex;
 
   &.hidden {
     display: none;
   }
 `;
 
-export const WrapFillet = styled.div`
-  max-width: 1440px;
+export const WraperFillet = styled.section`
```
Comment (renatoam) [inline src/modules/shared/components/molecules/Fillet/styles.ts:28]: Wrapper, mas já comentei ali em cima

File: src/modules/shared/constants/fillet.tsx [hunk 11]
```diff
@@ -0,0 +1,27 @@
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import DateRangeIcon from "@mui/icons-material/DateRange";
+import PhoneIcon from "@mui/icons-material/Phone";
+
+interface textFillet {
```
Comment (renatoam) [inline src/modules/shared/constants/fillet.tsx:5]: Types e Interface devem iniciar com letra maiúscula.

**Review by renatoam — CHANGES_REQUESTED** [review]
Somente nomenclaturas

**Review by renatoam — APPROVED** [review]
Done!

**Review by renatoam — DISMISSED** [review]

**Review by renatoam — APPROVED** [review]

**Comment by renatoam** [comment]
# [Codecov](https://codecov.io/gh/renatoam/amc-services/pull/19?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo) Report
> Merging [#19](https://codecov.io/gh/renatoam/amc-services/pull/19?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo) (9dd336c) into [develop](https://codecov.io/gh/renatoam/amc-services/commit/631a277b4d514ae0d2296a6b6673463073689fc4?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo) (631a277) will **not change** coverage.
> The diff coverage is `100.00%`.

```diff
@@            Coverage Diff            @@
##           develop       #19   +/-   ##
=========================================
  Coverage   100.00%   100.00%           
=========================================
  Files            1        13   +12     
  Lines            1        55   +54     
  Branches         0         3    +3     
=========================================
+ Hits             1        55   +54     
```


| [Impacted Files](https://codecov.io/gh/renatoam/amc-services/pull/19?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo) | Coverage Δ | |
|---|---|---|
| [...rc/modules/shared/components/atoms/Image/index.tsx](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvSW1hZ2UvaW5kZXgudHN4) | `100.00% <100.00%> (ø)` | |
| [...rc/modules/shared/components/atoms/Image/styles.ts](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvSW1hZ2Uvc3R5bGVzLnRz) | `100.00% <100.00%> (ø)` | |
| [src/modules/shared/components/atoms/Link/index.tsx](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvTGluay9pbmRleC50c3g=) | `100.00% <100.00%> (ø)` | |
| [src/modules/shared/components/atoms/List/index.tsx](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvTGlzdC9pbmRleC50c3g=) | `100.00% <100.00%> (ø)` | |
| [src/modules/shared/components/atoms/List/styles.ts](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvTGlzdC9zdHlsZXMudHM=) | `100.00% <100.00%> (ø)` | |
| [...modules/shared/components/atoms/ListItem/index.tsx](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvTGlzdEl0ZW0vaW5kZXgudHN4) | `100.00% <100.00%> (ø)` | |
| [...modules/shared/components/atoms/ListItem/styles.ts](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvTGlzdEl0ZW0vc3R5bGVzLnRz) | `100.00% <100.00%> (ø)` | |
| [src/modules/shared/components/atoms/Text/index.tsx](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvVGV4dC9pbmRleC50c3g=) | `100.00% <100.00%> (ø)` | |
| [src/modules/shared/components/atoms/Text/styles.ts](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvYXRvbXMvVGV4dC9zdHlsZXMudHM=) | `100.00% <100.00%> (ø)` | |
| [...dules/shared/components/molecules/Header/index.tsx](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo#diff-c3JjL21vZHVsZXMvc2hhcmVkL2NvbXBvbmVudHMvbW9sZWN1bGVzL0hlYWRlci9pbmRleC50c3g=) | `100.00% <100.00%> (ø)` | |
| ... and [4 more](https://codecov.io/gh/renatoam/amc-services/pull/19/diff?src=pr&el=tree-more&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/renatoam/amc-services/pull/19?src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/renatoam/amc-services/pull/19?src=pr&el=footer&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo). Last update [68802bb...9dd336c](https://codecov.io/gh/renatoam/amc-services/pull/19?src=pr&el=lastupdated&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Renato+Melo).

**Pull request merged**